packages/nodejs/src/abby/helpers.ts

```diff
@@ -0,0 +1,15 @@
+import { Request } from "express";
+
+/**
+ * helper function to parse express cookie
+ * @param req express Request
+ */
+export function parseCookies(req: Request) {
+    const cookies = req.headers.cookie;
+    const cookieMap = new Map<string, string>();
+    cookies?.split(";").map((res, index) => {
```

Comment:
other packages use a lib for that? lets do it everywhere the same way. either add that package here too or introduce an internal util package for stuff like that